packages/teleport/src/lib/util.ts

```diff
@@ -32,3 +32,15 @@ export function pluralize(num: number, word: string) {
 
   return word;
 }
+
+// Adapted from https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto/digest#converting_a_digest_to_a_hex_string
+export async function Sha256Digest(
```

Comment:
Could you add a test for this plz.

Reply:
It's difficult because `window.crypto` is not currently supported by `jsdom`. I tried the node and external library suggested in [this thread](https://github.com/jsdom/jsdom/issues/1612), however neither solution had `crypto.subtle.digest`

e.g. I created a `packages/teleport/src/lib/util.test.ts` and tried both of
```typescript
import { Sha256Digest } from './util';

const crypto = require('crypto');
// eslint-disable-next-line jest/require-hook
Object.defineProperty(global.self, 'crypto', {
  value: Object.setPrototypeOf({ subtle: crypto.subtle }, crypto),
});

// This is needed for tests until jsdom adds support for TextEncoder (https://github.com/jsdom/jsdom/issues/2524)
const { TextEncoder: TestTextEncoder } = require('util');
window.TextEncoder = window.TextEncoder || TestTextEncoder;

test('the Sha256Digest function does something sensible', () => {
  let digest = Sha256Digest('digest me');
  expect(digest).toBe('d');
});

```

```typescript
import { Crypto } from '@peculiar/webcrypto';

import { Sha256Digest } from './util';

global.crypto = new Crypto();

const crypto = require('crypto');
// eslint-disable-next-line jest/require-hook
Object.defineProperty(global.self, 'crypto', {
  value: Object.setPrototypeOf({ subtle: crypto.subtle }, crypto),
});

// This is needed for tests until jsdom adds support for TextEncoder (https://github.com/jsdom/jsdom/issues/2524)
const { TextEncoder: TestTextEncoder } = require('util');
window.TextEncoder = window.TextEncoder || TestTextEncoder;

test('the Sha256Digest function does something sensible', () => {
  let digest = Sha256Digest('digest me');
  expect(digest).toBe('d');
});

```